pom.xml

```diff
@@ -84,5 +90,26 @@
             </plugin>
         </plugins>
     </build>
-
+    <repositories>
+        <repository>
+            <id>central</id>
+            <url>https://repo1.maven.org/maven2</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
```

Comment:
for å pirke kan du sikkert berre fjerne det med snapshots, skal vel aldri laste ned snapshot versjonar frå andre. På den andre sidan så er det sikkert "ingen" som lastar opp det heller...